tcms/core/widgets.py

```diff
@@ -40,3 +41,18 @@ class Media:
             "prismjs/plugins/autoloader/prism-autoloader.min.js",
             "js/simplemde_security_override.js",
         ]
+
+
+class DurationWidget(forms.Textarea):
+
+    def index(request, attrs=None):
+        duration_template = "tcms/templates/include/duration_picker_script.html"
```

Comment:
`duration_template` can be renamed to `template_name` and made a class attribute because the value doesn't change. 

Also the value should be `include/duration_picker_script.html` b/c Django searches for templates based on the `TEMPLATES` -> `DIRS` setting, see `settings/common.py`